server/src/test/scala/services/crunch/CrunchTimezoneSpec.scala

```diff
@@ -40,24 +40,24 @@ class CrunchTimezoneSpec extends CrunchTestLike {
     }
 
     "Min / Max desks in BST " >> {
-      "Given flights with one passenger and one split to eea desk " +
-        "When the date falls within BST " +
-        "Then I should see min desks allocated in alignment with BST" >> {
-        val minMaxDesks: Map[Terminal, Map[Queue, (List[Int], List[Int])]] = Map(
-          T1 -> Map(
-            Queues.EeaDesk -> Tuple2(0 :: 5 :: List.fill[Int](22)(0), List.fill[Int](24)(20)),
-            Queues.NonEeaDesk -> Tuple2(0 :: 5 :: List.fill[Int](22)(0), List.fill[Int](24)(20))
-          )
-        )
+      "Given flights with one passenger and one split to eea desk " >> {
+        "When the date falls within BST " >> {
+          "Then I should see min desks allocated in alignment with BST" >> {
+            val minMaxDesks: Map[Terminal, Map[Queue, (List[Int], List[Int])]] = Map(
+              T1 -> Map(
+                Queues.EeaDesk -> Tuple2(0 :: 5 :: List.fill[Int](22)(0), List.fill[Int](24)(20)),
+                Queues.NonEeaDesk -> Tuple2(0 :: 5 :: List.fill[Int](22)(0), List.fill[Int](24)(20))
+                )
+              )
 
-        val scheduled = "2017-06-01T00:00Z"
+            val scheduled = "2017-06-01T00:00Z" // 2017-06-01 01:00 BST
```

Comment:
do we still want this comment?

Reply:
removed and renamed variable to better reflect its bst value